Products/CMFPlone/interfaces/controlpanel.py

```diff
@@ -1457,6 +1458,12 @@ class IUserGroupsSettingsSchema(Interface):
     )
 
 
+def validate_twitter_username(value):
+    if value and value.startswith('@'):
+        raise Invalid(u'Twitter username should not include the "@" prefix character.')  # noqa: E501
```

Comment:
And actually while at it, why do you raise Invalid and rather just remove the ``@`` character if there is any, which would be more helpful.